frontend/packages/console-shared/src/components/spotlight/InteractiveSpotlight.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { PopperOptions } from 'popper.js';
+import { Popper } from '../popper';
+import './spotlight.scss';
+
+type InteractiveSpotlightProps = {
+  element: Element;
+};
+
+const InteractiveSpotlight: React.FC<InteractiveSpotlightProps> = ({ element }) => {
+  const { height, width } = element.getBoundingClientRect();
+  const style: React.CSSProperties = {
+    height,
+    width,
+  };
+  const popperOptions: PopperOptions = {
+    modifiers: {
+      preventOverflow: {
+        enabled: false,
+      },
+    },
+  };
+  const isInViewport = (elementToCheck: Element) => {
+    const rect = elementToCheck.getBoundingClientRect();
+    return (
+      rect.top >= 0 &&
+      rect.left >= 0 &&
+      rect.bottom <= (window.innerHeight || document.documentElement.clientHeight) &&
+      rect.right <= (window.innerWidth || document.documentElement.clientWidth)
+    );
+  };
+  const [clicked, setClicked] = React.useState(false);
+
+  React.useEffect(() => {
+    if (!isInViewport(element)) {
+      element.scrollIntoView();
+    }
+    const handleClick = () => setClicked(true);
+    document.addEventListener('mousedown', handleClick);
+    return () => {
+      document.removeEventListener('mousedown', handleClick);
+    };
```

Comment:
```suggestion
    const handleClick = () => setClicked(true);
    document.addEventListener('click', handleClick);
    return () => {
      document.removeEventListener('click', handleClick);
    };
```
Changing these to `click` will help support mobile. `mousedown` is a non touch screen event.

Reply:
done